distrho/src/DistrhoUI.cpp

```diff
@@ -27,6 +27,9 @@
 # include "src/TopLevelWidgetPrivateData.hpp"
 #endif
 
+#include "DistrhoUIPrivateData.hpp"
+#include "src/WindowPrivateData.hpp"
```

Comment:
hmm this WindowPrivateData is for DGL stuff, so it better not be in use for external UIs.
Can you move it inside the ifdef after TopLevelWidgetPrivateData ?
If it doesnt break the usual build, it will be good for merging.
Thanks!

Reply:
Sure, done. Both main build and EmbedExternalUI build work.

Reply:
cool, thanks very much